index.html

```diff
@@ -2758,10 +2758,21 @@ <h3>Terms of Use</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://www.w3.org/2018/credentials/examples/v1"
+    "https://www.w3.org/2018/credentials/examples/v1",
+    {
+        "VerifiablePresentationExtension": {
+          "@id": "https://www.w3.org/2018/credentials#VerifiablePresentationExtension",
```

Comment:
```suggestion
          "@id": "https://www.w3.org/2018/credentials/examples#VerifiablePresentationTermsOfUseExtension",
```

We should be wary of putting hacks in the credentials vocabulary.

Reply:
ahh yes, good catch. I didn't mean to put it in that vocabulary. I'll be more careful about that.